samples/core/src/main/java/io/oasp/gastronomy/restaurant/offermanagement/service/impl/rest/OffermanagementRestServiceImpl.java

```diff
@@ -300,22 +282,22 @@ public void deleteProduct(@PathParam("id") long id) {
    * @param sortBy sorting specification
    * @return list with all {@link ProductEto}s that match the {@link ProductFilter} criteria
    */
-  @GET
-  @Path("/product/sortby/{sortBy}")
+  @Override
   @Deprecated
   public List<ProductEto> getFilteredProducts(ProductFilter productFilter, @PathParam("sortBy") ProductSortBy sortBy) {
 
     return this.offermanagement.findProductsFiltered(productFilter, sortBy);
   }
 
+  @Override
   @SuppressWarnings("javadoc")
   @Consumes("multipart/mixed")
   @POST
   @Path("/product/{id}/picture")
   public void updateProductPicture(@PathParam("id") long productId,
```

Comment:
Also remove the redundant JAX-RS annotations from the implementation after they have been moved to the interface.